lib/pages/add_bridge/add_bridge.dart

```diff
@@ -207,58 +207,37 @@ class BotController extends State<AddBridge> {
 
     final response = await dio.get('${network.apiPath}/provision/v3/whoami?user_id=$userId');
 
-    interpretBridgeResponse(response, network);
+    final status = interpretBridgeResponse(response);
+
+    switch (status) {
+      case ConnectionStatus.connected:
+        setState(() => network.updateConnectionResult(true));
+        break;
+      case ConnectionStatus.notConnected:
+        setState(() => network.updateConnectionResult(false));
+        break;
+      case ConnectionStatus.error:
+        _handleError(network, ConnectionError.unknown);
```

Comment:
This gives no information to the user as to what went wrong. Consider setting an error message, if appropriate